internal/pkg/service/buffer/worker/service/file_swap.go

```diff
@@ -0,0 +1,83 @@
+package service
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/keboola/go-client/pkg/keboola"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/file"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/rollback"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/task"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+const (
+	fileSwapTaskType = "file.swap"
+)
+
+func (s *Service) swapFile(fileKey key.FileKey, reason string) (err error) {
```

Comment:
Jediny rozdiel je, ze tieto operacie su definovane nad `Service`, namiesto `checker`, teda o uroven vyssie.

Spravil som to tak, aby to bolo rovnako ako pri ostatnych operaciach.

Rozdiel pri tychto 2 operaciach je v tom, ze nie su triggrovane nejakou zmenou v etcd, ale vyhodnotenim upload/import `conditions`, ktore sa vyhodnocuju pravidelne.

Tieto 2 operacie vlastne spustaju cely workflow ohladom slice upload / file import, a to tak, ze vytvoria novu slice/subor do ktorého sa majú nahrávať nové dáta.

checker embedduje `Service`:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/565b2689-249e-420c-adef-2304c23f94e5)